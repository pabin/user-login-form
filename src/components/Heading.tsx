import React from 'react';
import { HEADING_DESCRIPTION, HEADING_TITLE } from '../constants/heading';
import styles from './styles/heading.module.css';

const Heading  = () => (
  <div className={styles.container}>
    <p className={styles.title}>{ HEADING_TITLE }</p>
    <p className={styles.description}>{ HEADING_DESCRIPTION }</p>
  </div>
)

export default Heading;