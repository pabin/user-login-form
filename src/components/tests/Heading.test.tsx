import React from 'react';
import { render } from '@testing-library/react'
import Heading from '../Heading';

describe('Heading', () => {
  test('renders Heading component correctly', async () => {
    const wrapper = render(<Heading />);
    expect(wrapper.baseElement).toMatchSnapshot();
  });
});