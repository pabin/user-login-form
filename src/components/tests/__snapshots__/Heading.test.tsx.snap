// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Heading renders Heading component correctly 1`] = `
<body>
  <div>
    <div
      class="container"
    >
      <p
        class="title"
      >
        Dummy Heading
      </p>
      <p
        class="description"
      >
        Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore at dolore magna aliqua
      </p>
    </div>
  </div>
</body>
`;
